data/cncf.yaml

```diff
@@ -45,6 +51,12 @@
       check_sets:
         - community
         - code
+        - docs
+    - name: website
+      url: https://github.com/antrea-io/website
+      check_sets:
+        - docs
+        - community
```

Comment:
There is already one repository with the `community` check set in this project, please remove this check set from the `website` repository.